docs/api/activity.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Activity"
-summary: "API endpoints for Activity."
+summary: |
```

Comment:
just for my own knowledge -- what does the upright slash do?

Reply:
https://yaml.org/spec/1.2.2/#:~:text=Shoes%0A%20%20quantity%3A%201-,2.3.%20Scalars,-Scalar%20content%20can

Probably not necessary now that I'm losing the line breaks to format descriptions as a single line.